tests/ha/test_control_pod_restart.py

```diff
@@ -729,3 +731,312 @@ def test_mpu_after_ctrl_pod_failover(self):
+        else:
+            assert_utils.assert_true(False, "IAM user CRUD operations are expected to be failed "
+                                            "during control pod failover")
+
+        LOGGER.info("Checking background process for bucket CRUD operations")
+        bkt_resp = tuple()
+        while len(bkt_resp) != 2:
+            bkt_resp = bkt_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not bkt_resp:
+            assert_utils.assert_true(False, "Background process failed to do bucket CRUD "
+                                            "operations")
+        exp_fail = bkt_resp[0]
+        failed = bkt_resp[1]
+        assert_utils.assert_false(len(exp_fail) or len(failed),
+                                  "Failures observed in background process for bucket "
+                                  f"CRUD operations. \nIn-flight Failed buckets: {exp_fail}"
+                                  f"\nFailed buckets: {failed}")
+        LOGGER.info("Step 5: Successfully verifyied responses from background processes")
+
+        LOGGER.info("Step 6: Perform new IAM users(%s) and buckets(%s) creation/deletion in loop",
+                    num_users, num_bkts)
+        output = Queue()
+        args = {'user_crud': True, 'bkt_crud': True, 'num_users': num_users,
+                'num_bkts': num_bkts, 's3_obj': s3_obj, 'output': output}
+        self.ha_obj.iam_bucket_cruds(event, **args)
+        LOGGER.info("Checking responses for IAM user CRUD operations")
+        iam_resp = tuple()
+        while len(iam_resp) != 3:
+            iam_resp = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not iam_resp:
+            assert_utils.assert_true(False, "Failed to do IAM user CRUD operations")
+        exp_fail = iam_resp[0]
+        failed = iam_resp[1]
+        assert_utils.assert_false(len(exp_fail) or len(failed), "Failure in IAM user CRUD "
+                                                                "operations. \nFailed users: "
+                                                                f"\nexp_fail: {exp_fail} and "
+                                                                f"\nfailed: {failed}")
+
+        LOGGER.info("Checking responses for bucket CRUD operations")
+        bkt_resp = tuple()
+        while len(bkt_resp) != 2:
+            bkt_resp = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not bkt_resp:
+            assert_utils.assert_true(False, "Failed to do bucket CRUD operations")
+        exp_fail = bkt_resp[0]
+        failed = bkt_resp[1]
+        assert_utils.assert_false(len(exp_fail) or len(failed),
+                                  "Failures observed in bucket CRUD operations. "
+                                  f"\nFailed buckets: \nexp_fail: {exp_fail} and "
+                                  f"\nfailed: {failed}")
+        LOGGER.info("Step 6: Successfully created %s new IAM users and %s buckets in loop",
```

Comment:
creation/deletion or
Performed new IAM users(%s) and buckets(%s) creation/deletion *